tests/integration/targets/homebrew/tasks/formulae.yml

```diff
@@ -381,3 +381,55 @@
         - "terraform_install_result.msg == 'Package upgraded: hashicorp/tap/terraform'"
         - "terraform_install_result.changed_pkgs == ['hashicorp/tap/terraform']"
         - "terraform_install_result.unchanged_pkgs == []"
+
+# Test tap with no public fallback
+- block:
+  - name: Tap ascii-image-converter homebrew repository
+    community.general.homebrew_tap:
+      name: TheZoraiz/ascii-image-converter
+    become: true
+    become_user: "{{ brew_stat.stat.pw_name }}"
+
+  - name: Install ascii from full tap name
+    community.general.homebrew:
+      name: TheZoraiz/ascii-image-converter/ascii-image-converter
+      state: latest
+    register: ascii_install_result
+    become: true
+    become_user: "{{ brew_stat.stat.pw_name }}"
```

Comment:
How about repeating this task and making sure that the second invocation has no change? (Same below for removal.)

Reply:
Sure, great idea, I've just added this in my last commit